frontend/app-development/hooks/mutations/useDeleteDatamodelMutation.ts

```diff
@@ -32,6 +32,9 @@ export const useDeleteDatamodelMutation = () => {
       queryClient.removeQueries({
         queryKey: [QueryKey.JsonSchema, org, app, xsdPath],
       });
+      queryClient.removeQueries({
+        queryKey: [QueryKey.AppMetadataModelIds, org, app, xsdPath],
+      });
```

Comment:
This queryKey is used in `useAppMetadataModelIdsQuery` and it does not seem to need the `xsdPath ` param. 🤔

```suggestion
      await queryClient.invalidateQueries({ queryKey: [QueryKey.AppMetadataModelIds, org, app] });
```

Reply:
Wops, the xsdPath was not suppose to be there! 

My though for including it was to ensure that the dataModelIds from appMetadata was refetched when a datamodel was deleted so we will not list non-existing datamodels in the dropdown for selecting data type for a process task in processeditor. Will this implementation not do what I intended? 😅

Reply:
But does it need to be awaited? 🫣

Reply:
oh, nice catch, wrong copy/paste, no it does not need to be awaited 🙂👍